satellite/marketing/marketingweb/server.go

```diff
@@ -79,16 +99,23 @@ func NewServer(logger *zap.Logger, config Config, listener net.Listener) (*Serve
 	return s, nil
 }
 
-// ServeHTTP handles index request
-func (s *Server) ServeHTTP(w http.ResponseWriter, req *http.Request) {
+// Serves index page and renders offer and credits tables
```

Comment:
The comment still doesn't follow the convention. i.e.
```
// <funcname> ...
```

Reply:
This is for an older commit. This comment isn't present in latest.